### ISSUE TITLE
Unable to deploy contract after rejecting first transaction

### ISSUE THREAD

**Issue by churik** [status-im/open-bounty#329]
### Description

*Type*: Bug
*Summary*: when user adds `bounty` label for first time -> in order to confirm payment address test transaction is requested. If user rejects transaction on this stage, than adds some amount and tries to create new issue with `bounty` label -> no more test transaction is requested, so contract deployment 
for this user is blocked.

#### Expected behavior
test transaction is requested every time until it will be approved

#### Actual behavior
test transaction is requested only once

### Reproduction

- Open https://openbounty.status.im 
- Sign up
- Install Status open-bounty app (https://github.com/apps/status-open-bounty-app)
- Whitelist user
- Update `My payment details` - set new address (I set address from MetaMask)
- Add issue with `bounty` label
- Go to Metamask. New test tansaction for 0 ETH should appear.
- Reject transaction, so in history should appear
![rejected](https://user-images.githubusercontent.com/4557972/36665753-12988962-1af1-11e8-9099-da8a594a88ff.png)
- create new issue, add bounty label.

### Additional Information

* Ethereum wallet used: MetaMask
* Browser: Chrome  64
* Operating System: macOS HighSierra 10.13

**Comment by churik** [comment]
Well, hard to say. It was reproduced for me only one time and after this, I was completely blocked from posting a new bounty.
Because it was reproducible only on MainNet, I didn't try more.
And after a while (on 14 Apr) - I installed MetaMask wallet on the same way, and I wasn't asked about test transaction.
So, for now, I confused what it was.
Is it OK if I'll try one more time and close if I cannot reproduce?

**Comment by rcullito** [comment]
> Is it OK if I'll try one more time and close if I cannot reproduce?

Yes I think that makes sense! I was unable to reproduce either yesterday (on 16 April).

**Comment by rcullito** [comment]
@churik I ended up experiencing this same behavior during confirming payouts (rather than on contract deploy) so definitely a bug. nice find! will keep it in mind for something to work on after revocation.

**Comment by churik** [comment]
@rcullito superb, it is a bit tricky.
Thanks for keeping me updated!
